<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1319</width>
    <height>669</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>851</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ajouter fournisseur</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>851</width>
        <height>491</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
background:white;
}

QLabel{
color:white;
font-size:16px;
}

QPushButton:hover {
transform: scale(1.2); 
width : 250px;
color:#F3D617;
}
QPushButton{
color:white;
border-radius:15px;
}

QLineEdit
{
background:transparent;
border:none;
color:#a0a0a0;
font-size:16px;
border-bottom:1px solid #333;
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="age">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>220</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>age</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>150</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>nom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="prenom">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>150</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>prenom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="tel">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>220</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>tel</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="email">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>290</y>
         <width>361</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>email</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouterF">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>380</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#001F26;</string>
       </property>
       <property name="text">
        <string>ajouter fournisseur</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>90</y>
         <width>311</width>
         <height>331</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background:transparent;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>C:/Users/moham/OneDrive/Bureau/c++_logo.png</normaloff>C:/Users/moham/OneDrive/Bureau/c++_logo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>affichage/modifer/supprimer</string>
     </attribute>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>851</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
background:white;
}

QLabel{
color:white;
font-size:16px;
}

QPushButton:hover {
transform: scale(1,3); 
color:red;
}

QPushButton{
color:white;
border-radius:15px;
}

QPushButton::hover{
color:black;
border-radius:15px;
}

QTableWidget{
border:none;
}

</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>290</y>
         <width>351</width>
         <height>181</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
background:white;

}
QLabel{
color:white;
font-size:16px;
}
QPushButton
{
background:red;
border-radius:16px;
}
QPushButton:hover {
transform: scale(5); 
width : 250px;
color:#F3D617;
}
QPushButton{
color:white;
border-radius:15px;
}
QLineEdit
{
background:transparent;
border:none;
color:#a0a0a0;
font-size:16px;
border-bottom:1px solid #333;
}</string>
       </property>
       <property name="title">
        <string>supprimer fournisseur</string>
       </property>
       <widget class="QLineEdit" name="deleteID">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>80</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deleteFbtn">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>80</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#001F26;</string>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>240</y>
         <width>351</width>
         <height>241</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
background:white;

}
QLabel{
color:white;
font-size:16px;
}
QPushButton
{
background:red;
border-radius:16px;
}
QPushButton:hover {
transform: scale(5); 
width : 250px;
color:#F3D617;
}
QPushButton{
color:white;
border-radius:15px;
}
QLineEdit
{
background:transparent;
border:none;
color:#a0a0a0;
font-size:16px;
border-bottom:1px solid #333;
}</string>
       </property>
       <property name="title">
        <string>modifer fournisseur</string>
       </property>
       <widget class="QLineEdit" name="updateID">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>100</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QPushButton" name="updateBTN">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>180</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#001F26;</string>
        </property>
        <property name="text">
         <string>modifer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="nomUpdate">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>nom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="prenomUpdate">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>160</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>prenom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ageUpdate">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>80</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>age</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="telUpdate">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>120</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>tel</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="emailUpdate">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>200</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>email</string>
        </property>
       </widget>
      </widget>
      <widget class="QTableView" name="Ftable">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>771</width>
         <height>181</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>tri/chercher</string>
     </attribute>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>861</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
background:white;
}

QLabel{
color:white;
font-size:16px;
}

QPushButton:hover {
transform: scale(1,3); 
color:red;
}

QPushButton{
color:white;
border-radius:15px;
}

QPushButton::hover{
color:black;
border-radius:15px;
}

QTableWidget{
border:none;
}


</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>781</width>
         <height>181</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
background:white;

}
QLabel{
color:white;
font-size:16px;
}
QPushButton
{
background:red;
border-radius:16px;
}
QPushButton:hover {
transform: scale(5); 
width : 250px;
color:#F3D617;
}
QPushButton{
color:white;
border-radius:15px;
}
QLineEdit
{
background:transparent;
border:none;
color:#a0a0a0;
font-size:16px;
border-bottom:1px solid #333;
}

</string>
       </property>
       <property name="title">
        <string>TRI</string>
       </property>
       <widget class="QPushButton" name="pushButtonStatistiques">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>380</y>
          <width>271</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
background:#333;

}
QLabel{
color:white;
font-size:16px;
}
QPushButton
{
background:red;
border-radius:16px;
}
QPushButton:hover {
color:blue;
border-size:20px;
background:#333
}
QPushButton{
color:white;
border-radius:15px;
}
QLineEdit
{
background:transparent;
border:none;
color:#b0b0b0;
font-size:16px;
border-bottom:1px solid #fefff4;
}
QTableWidget{
background:transparent;
border:none;
}
QTableWidget:hover{
border:3px;
}
</string>
        </property>
        <property name="text">
         <string>STATISTIQUES</string>
        </property>
       </widget>
       <widget class="QComboBox" name="croissanceBox">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>50</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid;
border-color:rgb(0, 0, 0);
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
color : #001F26;
    margin: 0 0 0 0;</string>
        </property>
        <item>
         <property name="text">
          <string>croissante</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>decroissante</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="attributeBox">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid;
border-color:rgb(0, 0, 0);
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
color : #001F26;
    margin: 0 0 0 0;</string>
        </property>
        <item>
         <property name="text">
          <string>ID</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AGE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NOM</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="triBTN">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>100</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#001F26;</string>
        </property>
        <property name="text">
         <string>afficher</string>
        </property>
       </widget>
      </widget>
      <widget class="QTableView" name="TriTable">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>100</y>
         <width>431</width>
         <height>121</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>280</y>
         <width>781</width>
         <height>191</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
background:white;

}
QLabel{
color:white;
font-size:16px;
}
QPushButton
{
background:red;
border-radius:16px;
}
QPushButton:hover {
transform: scale(5); 
width : 250px;
color:#F3D617;
}
QPushButton{
color:white;
border-radius:15px;
}
QLineEdit
{
background:transparent;
border:none;
color:#a0a0a0;
font-size:16px;
border-bottom:1px solid #333;
}</string>
       </property>
       <property name="title">
        <string>Recherche par ID</string>
       </property>
       <widget class="QLineEdit" name="searchIDinput">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>90</y>
          <width>61</width>
          <height>22</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QPushButton" name="chercherID">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>90</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#001F26;</string>
        </property>
        <property name="text">
         <string>chercher</string>
        </property>
       </widget>
       <widget class="QTableView" name="searchTable">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>40</y>
          <width>501</width>
          <height>121</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>statistique</string>
     </attribute>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>861</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
background:white;
}

QLabel{
color:white;
font-size:16px;
}

QPushButton:hover {
transform: scale(1,3); 
color:red;
}

QPushButton{
color:white;
border-radius:15px;
}

QPushButton::hover{
color:black;
border-radius:15px;
}

QTableWidget{
border:none;
}


</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1319</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
